ci/bin/format.dart

```diff
@@ -314,11 +315,11 @@ class ClangFormatChecker extends FormatChecker {
   }) {
     /*late*/ String clangOs;
     if (Platform.isLinux) {
-      clangOs = 'linux-x64';
+      clangOs = Abi.current().toString().replaceFirst('_', '-');
     } else if (Platform.isMacOS) {
-      clangOs = 'mac-x64';
+      clangOs = 'mac-${Abi.current().toString().split('_')[1]}';
     } else if (Platform.isWindows) {
-      clangOs = 'windows-x64';
+      clangOs = Abi.current().toString().replaceFirst('_', '-');
```

Comment:
I would prefer if we had specified strings for each binary rather than processing the ABI string.

```
if (ABI.current() == Abi.macosx64) {
  clangOs = 'mac-amd64'; //
} else {
 //
}
```

Reply:
Alright, I've switched over to using that.